renovate.json

```diff
@@ -21,9 +21,15 @@
   "packageRules": [
     {
       "packagePatterns": ["^@angular/.*"],
+      "excludePackagePatterns": ["^@angular/dev-infra-private"],
       "groupName": "angular",
       "pinVersions": false
     },
+    {
+      "packagePatterns": ["^@angular/dev-infra-private"],
+      "groupName": "dev-infra",
+      "labels": ["action: merge", "target: patch"]
```

Comment:
I think this is not enough since likely the commit will not apply cleanly due to possible conflicts in package.json.

Reply:
That said, maybe we should try this approach first, and see how often we do have conflict before going with the other approach.

Thoughts?

Reply:
Lets try this first as I imagine that we don't actually run into conflicts very often because the dependencies don't often change for dev-infra.

Reply:
SGTM.

Reply:
It appears that as I suspected this approach will not work as the commit cannot be applied cleanly 

Example of another PR https://github.com/angular/angular-cli/pull/21409

Reply:
Let me know if we should explore the `baseBranch` option.